### PR TITLE
BROKER: Insert School

### PR DESCRIPTION
**Pull request by MahalakshmiMurari**
image

### CODE REVIEW

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Address review comments

File: SchoolApp.ConsoleApp/Brokers/Storages/StorageBroker.School.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace SchoolApp.ConsoleApp.Brokers.Storages
     internal partial class StorageBroker : IStorageBroker
     {
         List<School> Schools = new List<School>();
-        public School insertSchool(School school)
+        public School InsertSchool(School school)
```
Comment (ShriHumrudha) [inline SchoolApp.ConsoleApp/Brokers/Storages/StorageBroker.School.cs:13]: Need space

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: SchoolApp.ConsoleApp/Brokers/Storages/IStorageBroker.School.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using SchoolApp.ConsoleApp.Models.Schools;
+
+namespace SchoolApp.ConsoleApp.Brokers.Storages
+{
+    internal partial interface IStorageBroker
+    {
+        School insertSchool(School school);
```
[the remark this answers is not in the document]
Comment (MahalakshmiMurari) [reply SchoolApp.ConsoleApp/Brokers/Storages/IStorageBroker.School.cs]: Changed the method name

File: SchoolApp.ConsoleApp/Models/Schools/School.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using System;
+
+namespace SchoolApp.ConsoleApp.Models.Schools
+{
+    internal class School
+    {
+        public string SchoolName { get; set; }
```
[the remark this answers is not in the document]
Comment (MahalakshmiMurari) [reply SchoolApp.ConsoleApp/Models/Schools/School.cs]: As suggested, removed the extra attributes

File: SchoolApp.ConsoleApp/Brokers/Storages/StorageBroker.School.cs (hunk 1 above)
Comment (MahalakshmiMurari) [reply SchoolApp.ConsoleApp/Brokers/Storages/StorageBroker.School.cs:14]: Added Space

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**